OpenRA.Mods.Common/Activities/Air/FlyDistance.cs

```diff
@@ -0,0 +1,41 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Activities;
+using OpenRA.Mods.Common.Traits;
+
+namespace OpenRA.Mods.Common.Activities
+{
+	public class FlyDistance : Activity
```

Comment:
What do you think about calling this `FlyForwards` and having a second ctor that accepts an `int ticks` to replace `FlyTimed`?